src/pages/en/guides/markdown-content.md

```diff
@@ -406,76 +375,104 @@ const { Content } = Astro.props.post
+
+Any additional plugins you apply in your MDX config will be applied *after* your configured Markdown plugins, and will apply only to `.mdx` files. 
+
+This example applies [`remark-toc`](https://github.com/remarkjs/remark-toc) to Markdown *and* MDX, and [`rehype-accessible-emojis`](https://www.npmjs.com/package/rehype-accessible-emojis) to MDX only, while preserving Astro's default plugins:
+
+```js title="astro.config.mjs"
 import { defineConfig } from 'astro/config';
 import remarkToc from 'remark-toc';
 import { rehypeAccessibleEmojis } from 'rehype-accessible-emojis';
 
-export default defineConfig({
+export default {
   markdown: {
+    // Applied to .md and .mdx files
     remarkPlugins: [remarkToc],
-    rehypePlugins: [rehypeAccessibleEmojis],
-    // Preserve Astro's default plugins: GitHub-flavored Markdown and Smartypants
-    // default: false
+    //Preserves remark-gfm and remark-smartypants
     extendDefaultPlugins: true,
   },
+  integrations: [mdx({
+    // Applied to .mdx files only
+    rehypePlugins: [rehypeAccessibleEmojis],
+  })],
 }
 ```
 
-#### Remark-rehype options
+#### Example: Injecting frontmatter
 
-Markdown content is transformed into HTML through remark-rehype which has [a number of options](https://github.com/remarkjs/remark-rehype#options).
+You can add frontmatter properties to all of your Markdown and MDX files by using a [remark or rehype plugin](#markdown-plugins).
 
-You can use remark-rehype options in your config file like so:
+1. Append a `customProperty` to the `data.astro.frontmatter` property from your plugin's `file` argument:
 
-```js
-// astro.config.mjs
-export default {
-  markdown: {
-    remarkRehype: {
-      footnoteLabel: 'Catatan kaki',
-      footnoteBackLabel: 'Kembali ke konten',
-    },
-  },
-};
-```
-
-### Injecting frontmatter
-
-You may want to add frontmatter properties to your Markdown files programmatically. By using a [remark or rehype plugin](#markdown-plugins), you can generate these properties based on a file's contents.
-
-You can append to the `data.astro.frontmatter` property from your plugin's `file` argument like so:
-
-```js title="example-remark-plugin.mjs"
-export function exampleRemarkPlugin() {
-  // All remark and rehype plugins return a separate function
-  return function (tree, file) {
-    file.data.astro.frontmatter.customProperty = 'Generated property';
-  }
-}
-```
+    ```js title="example-remark-plugin.mjs"
+    export function exampleRemarkPlugin() {
+      // All remark and rehype plugins return a separate function
+      return function (tree, file) {
+        file.data.astro.frontmatter.customProperty = 'Generated property';
+      }
+    }
+    ```
 
-After applying this plugin to your `markdown` config:
+2. Apply this plugin to your `markdown`  or `mdx` integration config:
 
 ```js title="astro.config.mjs" "import { exampleRemarkPlugin } from './example-remark-plugin.mjs';" "remarkPlugins: [exampleRemarkPlugin],"
```

Comment:
Indentation needs fixing for this code sample, the “or,” and the next code sample so that they nest under 2. here.